android/src/main/java/com/swmansion/reanimated/NativeProxy.java

```diff
@@ -113,6 +118,25 @@ public Scheduler getScheduler() {
     return mScheduler;
   }
 
+  private void toggleSlowAnimations() {
+  }
```

Comment:
looks like it's not used

Reply:
I moved toggling logic there in be69639, it allowed using the method reference syntax to make `addCustomDevOption` a bit nicer to read